swagger.yml

```diff
@@ -378,6 +378,12 @@ definitions:
   EligibilityOverrideDTO:
     type: "object"
     properties:
+      childrenDob:
+        type: "array"
+        description: "The dates of birth of the claimant's children (if they have\
```

Comment:
Can we update the description to state that an empty list must be used if there are no children.